pom.xml

```diff
@@ -11,10 +11,12 @@
     </parent>
     <groupId>com.vaadin</groupId>
     <artifactId>vaadin-archetypes</artifactId>
-    <version>13.0.2</version>
+    <version>13.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
-
+    <properties>
+        <vaadin.version>13.0.2</vaadin.version>
```

Comment:
So now the release build has to also use `versions:update-properties` in addition to `versions:set`.